### PR TITLE
adds nomination dashboard filters

### PR DESCRIPTION
**Pull request by TheDancingClown**
https://drive.google.com/file/d/1soVr-PhHbDhAZdTBbjjOLqYNwhgfUYZM/view
https://app.asana.com/0/1200175839265057/1200311084223208

- Removes uppercase styling for filter labels
- Removes flag and urn column
- Splits ceremonial county column into assigned lieutenancy and ceremonial county from form
- Adds filters for new columns
- Fixes activity filter to display incomplete forms
- Adds copy help when no forms are checked

Screenshot 2021-08-05 at 10 36 41
Screenshot 2021-08-05 at 10 36 53
Screenshot 2021-08-05 at 10 37 05

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ if ($('.bulk-assignment-container').length > 0) {
 
     if (show_button) {
       $(".bulk-assignment-container").addClass("show-container")
-
+      $(".bulk-assignment-help").addClass("hide")
```
Comment (mauricioac) [inline app/javascript/packs/application.js:67]: use `govuk-!-display-none` instead of the `hide` class

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -73,6 +73,7 @@ if ($('.bulk-assignment-container').length > 0) {
       }
     } else {
       $(".bulk-assignment-container").removeClass("show-container")
+      $(".bulk-assignment-help").removeClass("hide")
```
Comment (mauricioac) [inline app/javascript/packs/application.js:76]: same here

File: app/javascript/packs/application.js [hunk 3]
```diff
@@ -83,11 +84,13 @@ if (dropdowns.length > 0) {
     var dropdown = $(this)[0];
     var multiselect = new CheckboxMultiselect(dropdown, {
       singleSelectionShowDirectly: true,
+      // includeBlank: true,
+      // placeholder: 'Select all',
```
Comment (mauricioac) [inline app/javascript/packs/application.js:88]: remove commented code

File: app/views/admin/form_answers/_bulk_assignment_modal.html.slim [hunk 4]
```diff
@@ -3,10 +3,10 @@
     .mm-modal__overlay tabindex="-1" data-micromodal-close=''
       .mm-modal__container role="dialog" aria-modal="true" aria-labelledby="modal-bulk-assign-lieutenants-title"
         header.mm-modal__header
-          h2.govuk-heading-l.mm-modal__title id="modal-bulk-assign-lieutenants-title"
+          h3.govuk-heading-l.mm-modal__title id="modal-bulk-assign-lieutenants-title"
```
Comment (mauricioac) [inline app/views/admin/form_answers/_bulk_assignment_modal.html.slim:6]: h3 should be `govuk-heading-m`

File: app/views/admin/form_answers/_bulk_assignment_modal.html.slim [hunk 5]
```diff
@@ -17,12 +17,11 @@
               = f.select :ceremonial_county_id,
                         CeremonialCounty.all.pluck(:name, :id),
                         { include_blank: true },
-                        class: "govuk-select"
+                        class: "govuk-select govuk-grid-column-one-half"
```
Comment (mauricioac) [inline app/views/admin/form_answers/_bulk_assignment_modal.html.slim:20]: Selects should be using grid layout. USe that on a wrapper div please

File: app/views/admin/form_answers/index.html.slim [hunk 6]
```diff
@@ -1,24 +1,29 @@
...
   | Nominations
 
-
 = simple_form_for @search, url: admin_form_answers_path, method: :get, as: :search, html: { class: 'search-form'} do |f|
   .govuk-grid-row
     .govuk-grid-column-one-quarter.if-js-hide
       = f.input :year, input_html: { value: params[:year] || @award_year.year }
     .govuk-grid-column-one-third
       .form-group.search-input
+        = f.label 'Search', class: 'govuk-visually-hidden'
```
Comment (mauricioac) [inline app/views/admin/form_answers/index.html.slim:10]: Make it visible please

**Review by mauricioac — CHANGES_REQUESTED** [review]

File: app/views/admin/form_answers/_bulk_assignment_modal.html.slim (hunk 5 above)
Comment (TheDancingClown) [reply app/views/admin/form_answers/_bulk_assignment_modal.html.slim]: Removed class to rely on autosizing

**Review by mauricioac — APPROVED** [review]

**Pull request merged**